addon/webextension/background/analytics.js

```diff
@@ -8,6 +8,33 @@ this.analytics = (function() {
   let telemetryPrefKnown = false;
   let telemetryPref;
 
+  function sendTiming(timingLabel, timingVar, timingValue) {
+    // sendTiming is only called in response to sendEvent, so no need to check
+    // the telemetry pref again here.
+    let timingCategory = "addon";
+    let url = main.getBackend() + "/timing";
+    let req = new XMLHttpRequest();
+    req.open("POST", url);
```

Comment:
We should probably start using `fetch()` (though I also keep falling back to XHR too ;)

Reply:
sure, already filed as #2261